crates/db_views/src/comment_view.rs

```diff
@@ -234,7 +239,7 @@ fn queries<'a>() -> Queries<
       // only order if filtering by a post id, or parent_path. DOS potential otherwise and max_depth + !post_id isn't used anyways (afaik)
       if options.post_id.is_some() || options.parent_path.is_some() {
         // Always order by the parent path first
-        query = query.order_by(subpath(comment::path, 0, -1));
+        query = query.then_order_by(subpath(comment::path, 0, -1));
```

Comment:
Oops, I just realized an issue with this. The comment tree needs to be in a proper hierarchical order. 

IE lets say a distinguished comment is in a branch, then it shouldn't come before the other top level ones. So this needs to be done in a way that orders by subpath then distinguished...

Reply:
Okay I moved it below that block, hope its okay now.